### PR TITLE
update to match bootstrap v3.0.3

### PR DESCRIPTION
**Pull request by m5o**
Update to match the latest bootstrap 3.0.3 release.

This is _my_ last update and contributing to this repository. Thank you @jlong for the trust and commit rights ;-)
In the early days I did the 1:1 sync by hand. Lastly the converter by @glebm did the main part translating the files from less to sass.

I guess for bootstrap 3.1.x there are big news if you like sass. The guys from [bootstrap-sass](https://github.com/thomas-mcdonald/bootstrap-sass) teamed up with [twbs](https://github.com/twbs/bootstrap).

The official [twbs/bootstrap-sass](https://github.com/twbs/bootstrap-sass) is here!

Happy Merry Christmas everyone! :christmas_tree: :santa: :gift: 

/m5o :sunny:

### CODE REVIEW

**Comment by jlong** [comment]
Thanks my friend! Excellent work. Should we update the README of the project to indicate that we now recommend  that you use twbs/bootstrap-sass instead?

**Comment by m5o** [comment]
Yeah good point. Please write that extra words. Would you shut down the repo completely?

**Comment by AJ-Acevedo** [comment]
Shutting down the repo could end up breaking many links, scripts and task runners like rake or grunt.
Some sort of deprecation notice could help.

**Comment by ryanburnette** [comment]
It's the end of an era.

To all the contributors, thanks for your work. It's helped so much!

**Comment by radum** [comment]
I wanted to update this with 3.0.3 and I first searched the pull requests to see if somebody did it and I saw this. Thanks @jlong, @m5o and all the contributors, I learned a lot from this one.

Could you still merge the pull request before we migrate to the official version?

File: bower.json [hunk 1]
```diff
@@ -1,9 +1,21 @@
 {
   "name": "sass-bootstrap",
-  "version": "3.0.2",
+  "version": "3.0.3",
   "main": ["./dist/js/bootstrap.js", "./dist/css/bootstrap.css", "./dist/fonts/*"],
   "ignore": [
-      "**/.*"
+      "**/.*",
+      "_*",
```
Comment (artwork-mpx) [inline bower.json:7]: Is it mistake? It removes all  _underscore files (_alerts.scss, _badges.scss, ...) from lib/.

**Comment by jlong** [comment]
Closing as there is now an official Sass Bootstrap port! Please add an issue there if this is still a problem: https://github.com/twbs/bootstrap-sass

**Pull request closed without merging**